src/python/bot/fuzzers/syzkaller/constants.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Constants that are meaningful to syzkaller.
+Should not have any dependancies.
+"""
+
+# syzkaller's exit code if a bug occurred in syzkaller.
+SYZKALLER_ERROR_EXITCODE = 1
+
+# syzkaller's exit code if a bug was found in the kernel code.
+TARGET_ERROR_EXITCODE = 77
```

Comment:
Are these exit codes correct/used ? these seem to be libfuzzer, skip for now if unused.